regression/contracts/function-calls-01/test-enf.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+--
+This confirms the accuracy of the postconditions.
```

Comment:
Could you elaborate a bit more? What do you mean by accuracy? For this test, we want to check the behavior of pre- and post-conditions using the `--enforce-all-contracts`, correct? In that case, we assume `requires` and assert `ensures`. It'd be helpful to document that here too. The same apply to all test descriptions.